README.md

```diff
@@ -19,20 +19,18 @@ Different modules of GLocal (left) and GLocal in action for large scale explorat
 # Papers
 If you find this package useful for your research, please consider citing our paper:
 
-* **Note:** Our paper was accepted for publication in IEEE RA-L, the information below will be updated upon publication. To read the paper please refer to ArXiv at the moment.
-
 * Lukas Schmid, Victor Reijgwart, Lionel Ott, Juan Nieto, Roland Siegwart, and Cesar Cadena, "**A Unified Approach for Autonomous Volumetric Exploration of Large Scale Environments under Severe Odometry Drift**", in *IEEE Robotics and Automation Letters*, 2021 \[[IEEE](https://ieeexplore.ieee.org/document/9387110) | [ArXiv](https://arxiv.org/abs/2010.09859) | Video\]
```

Comment:
Please add the vol, issue, and pages also here

Reply:
Thank you, good catch!